src/main/java/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel.java

```diff
@@ -0,0 +1,78 @@
+package org.jenkinsci.plugins.workflow.flow;
+
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.StaplerRequest;
+
+import javax.annotation.CheckForNull;
+
+/**
+ * Supports a global default durability level for users
+ * @author Sam Van Oort
+ */
+public class GlobalDefaultFlowDurabilityLevel extends AbstractDescribableImpl<GlobalDefaultFlowDurabilityLevel>  {
```

Comment:
@jglick Explain please.

Reply:
Extend `GlobalConfiguration` for global settings rather than rolling your own persistence & UI. See the Jenkins plugin archetype demonstrating its usage.